lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -823,14 +821,36 @@ def create_winrm_transport(machine_spec, machine_options, instance)
       private_key = OpenSSL::PKey::RSA.new(pem_bytes)
       decrypted_password = private_key.private_decrypt decoded
 
-      winrm_options = {
+      shared_winrm_options = {
         :user => machine_spec.reference['winrm_username'] || 'Administrator',
-        :pass => decrypted_password,
-        :disable_sspi => true,
-        :basic_auth_only => true
+        :pass => machine_options[:password] || decrypted_password
       }
 
-      Chef::Provisioning::Transport::WinRM.new("#{endpoint}", type, winrm_options, {})
+      if(winrm_transport_options['https'] || winrm_transport_options[:https]  )
```

Comment:
If `winrm_transport_options` comes from `machine_options[:winrm_transport]` then I think we should just use symbols everywhere instead of the mix of strings and symbols.

The reason we use strings in `machine_spec.reference` is that it is converting them to/from a data_bag.  Otherwise, symbols are prefered.